pkgs/development/python-modules/playwright/driver.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchzip
+, playwrightVersion
+}:
+
+stdenv.mkDerivation rec {
+    name = "playwright-driver";
+    src =
+    let
+    inherit (stdenv.hostPlatform) system;
+    selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
+    suffix = selectSystem {
+        x86_64-linux = "linux";
+        aarch64-linux = "linux-arm64";
+        x86_64-darwin = "mac";
+        aarch64-darwin = "mac-arm64";
+    };
+    sha256 = selectSystem {
+        x86_64-linux = "sha256-jHsxZMch8FNRgMvoLNJEgRWCbufe5SgD6WX6CPq9wgg=";
+        aarch64-linux = "sha256-XHiKYQHVcRPWa2fIfBr7k8Vu6vFz2x4N9j/pU4Y3g0s=";
+        x86_64-darwin = "sha256-SbyAsIO2lUiS8HUu6ddVIMIi6E0mm3w9SzInaGyH4Ws=";
+        aarch64-darwin = "sha256-SeEhB4cq4vo6sT1DiuBj339HGKkMK8erzY3vIjs95oQ=";
+    };
+    in
```

Comment:
`nixpkgs-fmt`, and perhaps move this to L5 for more readability at the cost of scope correctness? It's weird to see a `src` attr this big, call's yours though.

Reply:
I've run nixpkgs-fmt on this, but as for the src attr, it was entirely based on the preexisting one here: https://github.com/NixOS/nixpkgs/blob/4848f9f3a622aec4364545ae2bb0197dbe7235c1/pkgs/tools/networking/boundary/default.nix#L7-L27